source/whats-new.txt

```diff
@@ -22,6 +22,14 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _upcoming-breaking-chages:
+
+Upcoming Breaking Changes
+-------------------------
+
+- Starting from an upcoming release, the driver will not support MongoDB
```

Comment:
Issue: The phrase "Starting from" doesn't strike my ear as right.

Suggestion: "Starting in" or "Starting with". 

Or reword entirely. E.g.: "In an upcoming release, the driver will drop support for MongoDB v3.4 and earlier."